pkg/cloudprovider/providers/aws/aws_routes.go

```diff
@@ -20,9 +20,11 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/golang/glog"
+
```

Comment:
why leave the blank lines?

Reply:
agreed, blank new lines between `github.com/*` aren't needed